src/checksum/zseries/crc32le-vx.sx

```diff
@@ -279,11 +279,13 @@ crc32_le_vgfm_generic:
 	VLM     %v8,%v15,0,%r15
 	lmg     %r14,%r15,240(%r15)
 	br	%r14
+#endif
 
 /*
  * Make sure the stack isn't executable with GCC (regardless of platform).
+ * Note: this section must always be output and can never be #ifdef'd out as
+ * part of conditional compilation.
  */
 #ifndef __clang__
```

Comment:
Not to be addressed in this review, but I wonder a bit. This file makes the GNU-stack note conditional on not using clang, the other makes it conditional on using ELF. I wonder which is right. Something for another day.

Reply:
Good catch, thanks -- I'll use `__ELF__`, afaict, that's accepted practice.